xrviz/fields.py

```diff
@@ -241,8 +242,12 @@ def guess_x_y(self, var):
         .. _`metpy.parse_cf`: https://github.com/Unidata/MetPy/blob/master/metpy/xarray.py#L335
         """
         try:
+            import metpy
             parsed_var = self.data.metpy.parse_cf(var)
-            x, y = parsed_var.metpy.coordinates('x', 'y')
+            if(Version(metpy.__version__)) > Version("0.12"):
+                x, y = parsed_var.metpy.longitude, parsed_var.metpy.latitude 
+            else:
+                x, y = parsed_var.metpy.coordinates('x', 'y') 
```

Comment:
So this doesn't work for 0.12, but it still gives a warning? In that case, we should explicitly suppress the warning here. 
```
with warnings.catch_warnings():
    warnings.filterwarnings("ignore", category=DeprecationWarning)
    x, y, = ...
```